test.ts

```diff
@@ -0,0 +1,45 @@
+import { NetworkNames, Sdk, EnvNames } from "./src";
+// import { Framework } from "@superfluid-finance/sdk-core";
+// import { ethers, BigNumber } from "ethers";
+
+
+async function main() {
+    try {
+        // let provider = new ethers.providers.JsonRpcProvider('https://rinkeby.infura.io/v3/502a8e83d34d471596ca857eec9771ff')
+        // const provider = new ethers.providers.JsonRpcProvider('https://rpc-mumbai.maticvigil.com')
+        const sdk = new Sdk(
+        // "0x513a984bbd054d9fb6d8ba656183185f55bad24a8f900a57a820077374fa9779",
+            "0x9648a8add89fd006b4d5a8f913b6547ffab680fa3b0bcd91247cc23092e47fd0",
```

Comment:
Is this file relevant? Why is it part of the source code?

Reply:
no just used for testing